README.md

```diff
@@ -20,31 +20,24 @@ Welcome to the Skulpt developer community! Check out the ideas list below. And t
 
-6. Expand the skulpt standard library to include more modules from the CPython standard library.  So far we have math, random, turtle, time (partial) random (partial) urllib (partial) unittest, image, DOM (partial) and re (partial).  Any of the partial modules could be completed, or many other CPython modules could be added.  Potential new modules from the standard library include:  functools, itertools, collections, datetime, operator, and string.  Many of these would be relatively easy projects for a less experienced student to take on.
+1. Expand the skulpt standard library to include more modules from the CPython standard library.  So far we have math, random, turtle, time (partial) random (partial) urllib (partial) unittest, image, DOM (partial) and re (partial).  Any of the partial modules could be completed, or many other CPython modules could be added.  Potential new modules from the standard library include:  functools, itertools, collections, datetime, operator, and string.  Many of these would be relatively easy projects for a less experienced student to take on.
 
-7. Over time we have had numerous requests for more advanced Python modules to be included in Skulpt.  These include, portions of matplotlib, tkinter, and numpy.  These are much more challenging because they contain C code in their implementation, but if a reasonable subset could be implemented in Javascript this would make it much easier to directly add many more python modules that rely on these three.  In addition, it would allow for skulpt to potentially be used in teaching an even broader set of topics.
+2. Over time we have had numerous requests for more advanced Python modules to be included in Skulpt.  These include, portions of matplotlib, tkinter, and numpy.  These are much more challenging because they contain C code in their implementation, but if a reasonable subset could be implemented in Javascript this would make it much easier to directly add many more python modules that rely on these three.  In addition, it would allow for skulpt to potentially be used in teaching an even broader set of topics.
 
-5. Expand and clean up the foreign function API.  This API is critical for implementing parts of the standard library.
+3. Expand and clean up the foreign function API.  This API is critical for implementing parts of the standard library.
 
+4. Currently builtin types (list, tuple, string, etc) are not subclassable.  Making the builtins subclassable would eliminate several known bugs in Skulpt.
 
-1. Currently builtin types (list, tuple, string, etc) are not subclassable.  Making the builtins subclassable would eliminate several known bugs in Skulpt.
+5. Implement decorators.  This would enable a whole bunch of pure python modules to be added to skulpt, plus allow us to add things like properties in a nice way.
```

Comment:
Actually, skulpt already has some basic decorator support. See https://github.com/skulpt/skulpt/pull/593.

@albertjan Are there any limitations, e.g. multiple decorators not supported?

Reply:
Just confirm? I'll change that for sure.

Reply:
I have no idea I've never tried to decorators 😄 one definitely works. and with #736 this more and more of those use cases will work.